apps/hash-ai-worker-ts/src/activities/flow-activities/research-entities-action.ts

```diff
@@ -248,7 +301,7 @@ export const researchEntitiesAction: FlowActionActivity = async ({
               };
             }
 
-            const status = await inferEntitiesFromWebPageWorkerAgent({
+            const status = await inferFactsFromWebPageWorkerAgent({
               prompt: inferencePrompt,
               entityTypes: input.entityTypes.filter(({ $id }) =>
                 entityTypeIds.includes($id),
```

Comment:
an error below should read 'facts' instead of entities'

Reply:
Thanks for spotting, addressed in https://github.com/hashintel/hash/pull/4497/commits/688e52a2913ec76f28e7cf2c1c61f544c0909d8f